src/Symfony/Component/HttpKernel/EventListener/ExceptionListener.php

```diff
@@ -30,11 +31,15 @@ class ExceptionListener implements EventSubscriberInterface
 {
     protected $controller;
     protected $logger;
+    protected $httpStatusCodeLogLevel = array();
 
-    public function __construct($controller, LoggerInterface $logger = null)
+    public function __construct($controller, LoggerInterface $logger = null, $httpStatusCodeLogLevel = array())
```

Comment:
missing array typehint

Reply:
ah cool, I wasn't sure if we we needed lower php compatibility